site/content/en/docs/Reference/kustomize_cli.md

```diff
@@ -1,8 +1,77 @@
 ---
-title: "kustomize CLI"
-linkTitle: "kustomize CLI"
+title: "Command line tool (kustomize)"
+linkTitle: "Command line tool (kustomize)"
```

Comment:
I personlly think that the `tool` and `(kustomize)` part could be omitted.
The [kubectl CLI](https://kubernetes.io/docs/reference/kubectl/) page introduces a new tool called `kubectl` in the kubernetes Reference docs, so it's useful information. 
But right now we are already in the Kustomize document site, so it feels sort of redundant. 

```suggestion
title: "Command Line Interface"
linkTitle: "Command Line Interface"
```

<img width="1156" alt="image" src="https://github.com/kubernetes-sigs/kustomize/assets/73531614/ad104b40-8400-4087-af94-a2dd5072092d">

Reply:
Updated title to "Command Line Interface"